assets/base.css

```diff
@@ -3,96 +3,65 @@
 :root,
 .color-background-1 {
   --color-foreground: var(--color-base-text);
-  --color-foreground-rgb: var(--color-base-text-rgb);
   --color-background: var(--color-base-background-1);
-  --color-background-rbg: var(--color-base-background-1-rgb);
-  --payment-terms-background-color: var(--color-base-background-1);
```

Comment:
Anyone know where this was being used?

Reply:
ah It might be for installments 🤔  Cause I remember there was something specific that was needed for the icon used in the payment terms to change itself and keep a good color contrast. So even though it looks like it's not used anywhere in the theme, it is by the installment API. 

To test it, you just need to change the currency in the test store to USD and the installments' info will show up. Then try and change the background color to a similar purple of the icon and it should automatically change.

Reply:
Essentially what Ludo said—here's some additional context for what we're checking for:
1. Check the parent element of the banner for a CSS variable called --payment-terms-background-color
2. Check the document root for a CSS variable called --payment-terms-background-color
3. Check the document root for a CSS variable called --color-body
4. If none are present the logo will default to purple

Something I completely forgot about though is that the CSS custom properties above _need_ to be specified in hex, so let's move this custom property to all layout files and update to `--payment-terms-background-color: {{ settings.colors_background_1 }}`.